www/docs/src/content/docs/0.1.0-beta.5/index.mdx

```diff
@@ -0,0 +1,28 @@
+---
+title: Welcome to Astro StudioCMS
+description: Get started building with Astro StudioCMS.
+template: splash
+banner:
+  content: |
+    We've renamed the package to <b>studiocms</b> 🎉
```

Comment:
I would prefer this to use `<strong>` tags

Reply:
This is the old version of the docs.... (beta.5)

Reply:
Updated the Latest version of the docs on [d869f86](https://github.com/astrolicious/studiocms/pull/281/commits/d869f869670c49fbd9c8f357a32ed15fe00a20c5)